CondFormats/GeometryObjects/interface/HcalParameters.h

```diff
@@ -64,6 +64,7 @@ class HcalParameters {
   std::vector<int>    phigroup;
   std::vector<LayerItem> layerGroupEtaSim, layerGroupEtaRec;
   int                 topologyMode;
+  int                 triggerMode;
```

Comment:
@bsunanda - this will require schema change. I'd suggest to use the topologyMode integer with mask to store/retrieve triggerMode.